.gitignore

```diff
@@ -42,3 +42,7 @@ profile.out
 
 # mobile related files
 cmd/aries-agent-mobile/build/
+
+# node.js modules
+node_modules/
```

Comment:
I think the directories containing package.json already have a .gitignore with node_modules. Although it doesn't hurt to also make it a global ignore.

Reply:
Yes , sir !
Because after I compiled, node_modules appeared in the root directory of the project, so I wanted to say whether it is possible to add node_modules to the global .gitignore . Obviously my thinking is too simple, thank you very much for your reply ! ^^